sktime/forecasting/base/adapters/_statsmodels.py

```diff
@@ -16,7 +16,7 @@ class _StatsModelsAdapter(BaseForecaster):
 
     _fitted_param_names = ()
     _tags = {
-        "univariate-only": True,
+        "univariate-only": False,
```

Comment:
this will change the tag for all descendants, not just the VAR forecaster! I don't think you should do this.